package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "eslint-plugin-wingsui-rules",
+  "version": "0.0.1",
+  "description": "A repository to host all the custom lint rules for wingsui repo",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wings-software/wingsui-eslint-rules.git"
```

Comment:
can we get the repo name and package name to match?

Reply:
package name follows the eslint convention : https://eslint.org/docs/developer-guide/working-with-plugins

Kept a different repo name as the eslint one was too verbose.